src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -85,6 +85,7 @@ internal static class EnforceOnBuildValues
         public const EnforceOnBuild RemoveRedundantNullableDirective = /*IDE0240*/ EnforceOnBuild.Recommended;
         public const EnforceOnBuild RemoveUnnecessaryNullableDirective = /*IDE0241*/ EnforceOnBuild.Recommended;
         public const EnforceOnBuild MakeStructReadOnly = /*IDE0250*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild UsePatternMatchingAsAndMemberAccess = /*IDE0260*/ EnforceOnBuild.Recommended;
```

Comment:
I am not familiar with this list - why do we need an explicit set of analyzers that are enforced on build?